### ISSUE TITLE
Broken URL in install_leofs.yml

### ISSUE THREAD

**Issue by mechanicalapple** [leo-project/leofs_ansible#2]
I've got an error during install playbook execution

```
anikeev@leo0:~/leofs_ansible$ ansible-playbook -i hosts install_leofs.yml --ask-sudo-pass
...
fatal: [leo3]: FAILED! => {"changed": false, "dest": "/tmp/leofs_1.4.0_ubuntu-14.04_amd64.deb", "msg": "Request failed", "response": "HTTP Error 404: Not Found", "state": "absent", "status_code": 404, "url": "https://github.com/leo-project/leofs/releases/download/1.4.0/leofs_1.4.0-1_ubuntu-14.04_amd64.deb"}
...
```

Sytem:
```
anikeev@leo0:~/leofs_ansible$ uname -a
Linux leo0 4.4.0-116-generic #140-Ubuntu SMP Mon Feb 12 21:23:04 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
anikeev@leo0:~/leofs_ansible$ dpkg -l | grep ansible
ii  ansible                             2.5.0-1ppa~xenial                          all          Ansible IT Automation
anikeev@leo0:~/leofs_ansible$ git log
commit a1eeeb00c6b3673aa0e41b5c7f1ca20176786375
Author: Wilson Li <windkithk@gmail.com>
Date:   Thu Dec 14 09:56:59 2017 +0900

    [Storage] Added Backend DB Fadvise WILLNEED Option

...
anikeev@leo0:~/leofs_ansible$ cat hosts
# Please check roles/common/vars/leofs_releases for available versions
[all:vars]
leofs_version=1.4.0
build_temp_path="/tmp/leofs_builder"
build_install_path="/tmp/"
build_branch="master"
source="package"

[builder]
leo1

# nodename of leo_manager_0 and leo_manager_1 are set at group_vars/all
[leo_manager_0]
leo1 

# nodename of leo_manager_0 and leo_manager_1 are set at group_vars/all
[leo_manager_1]
leo2 

[leo_storage]
leo3 leofs_module_nodename=S0@leo3
leo4 leofs_module_nodename=S1@leo4

[leo_gateway]
leo5 leofs_module_nodename=G0@leo5

[leofs_nodes:children]
leo_manager_0
leo_manager_1
leo_gateway
leo_storage
```

**Comment by yosukehara** [comment]
@mechanicalapple I've recognized [the name of the latest package for Ubuntu](https://github.com/leo-project/leofs/releases/tag/1.4.0) does not match [package_Ubuntu_14.04.yml](https://github.com/leo-project/leofs_ansible/blob/master/roles/common/vars/package_Ubuntu_14.04.yml). I'll fix it.

**Comment by yosukehara** [comment]
I've fixed the package url of LeoFS v1.4.0 for Ubuntu as follows:

- [leofs_1.4.0-1_ubuntu-14.04_amd64.deb](https://github.com/leo-project/leofs/releases/download/1.4.0/leofs_1.4.0-1_ubuntu-14.04_amd64.deb)

Do you think you could retry that? If it is successful, close this issue.

**Comment by mechanicalapple** [comment]
Yes, it works. Thanks!
